### DISCUSSION TITLE
OPEN Alliance Automotive Ethernet ECU Test v3.0/2.0 SOMEIP

### DISCUSSION THREAD

**Discussion by bfzravi** [COVESA/vsomeip#632] [Q&A]
Hi ,
Could you please share some sample code of SOMEIP ETS which satisfies the "OPEN Alliance Automotive Ethernet ECU Test Specification" . 

Thank you very much!

**Comment by goncaloalmeida** [comment]
@bfzravi we will discuss this internally if it is possible or not.
in a future a release we are planning to provide at least the base library of the tests. the tests itself only if we receive authorization.
